checks/check_crl

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
+# author: Oguzhan Cicek, OpenSource Security GmbH - oguzhan(at)os-s.de
+
+
+def check_crl_arguments(params):
+    return "-u {} -w {} -c {}".format(params.get("url"), params.get("minutes")[0], params.get("minutes")[1])
```

Comment:
This line is incorrect: if `"minutes"` is not in `params`, you will end up with the expression `None[0]`, which will of course crash. This should never happen, since the valuespec  below has no optional keys, so please use `params["url"]` etc.